### PR TITLE
New Resource: azurerm_securitycenter_workspace

### PR DESCRIPTION
**Pull request by katbyte**

### CODE REVIEW

**Comment by katbyte** [comment]
#2043 and #2045 should be merged first

File: azurerm/resource_arm_security_center_workspace.go [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+	"log"
+	"time"
+)
+
+//only valid name is default
+// Message="Invalid workspace settings name 'kttest' , only default is allowed "
+const resourceArmSecurityCenterWorkspaceName = "default"
```
Comment (tombuildsstuff) [inline azurerm/resource_arm_security_center_workspace.go]: can we ditch `resourceArm` from the start of this?

File: website/docs/r/security_center_workspace.markdown [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  scope        = "/subscriptions/00000000-0000-0000-0000-000000000000"
+  workspace_id = "${azurerm_log_analytics_workspace.example.id}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `scope` - (Required) The scope of VMs to send their security data to the desired workspace, unless overridden by a setting with more specific scope.
```
Comment (tombuildsstuff) [inline website/docs/r/security_center_workspace.markdown]: is this just VM's, or is this Resources?

File: website/docs/r/security_center_workspace.markdown [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+The following attributes are exported:
+
+* `id` - The Security Center Contact ID.
+
+## Import
+
+The contact can be imported using the `resource id`, e.g.
+
+```shell
+terraform import azurerm_securitycenter_workspace.example /subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Security/securityWorkspaces/default
```
Comment (tombuildsstuff) [inline website/docs/r/security_center_workspace.markdown]: `securitycenter` -> `security_center`

File: website/docs/r/security_center_workspace.markdown (hunk 2 above)
Comment (katbyte) [reply website/docs/r/security_center_workspace.markdown]: The API docs specifically said VMs

**Pull request merged**